lang/ui.zh-cn.json

```diff
@@ -28,7 +28,7 @@
     "description": "Change made to file"
   },
   "api-description": {
-    "defaultMessage": "关于用法和示例，请参阅 <link>Reference</link>",
```

Comment:
These is the only other translation change and it seems like a simple win to include it.